deploy/kubernetes/webhook-example/create-cert.sh

```diff
@@ -79,18 +79,19 @@ kubectl delete csr ${csrName} 2>/dev/null || true
 
 # create  server cert/key CSR and  send to k8s API
 cat <<EOF | kubectl create -f -
-apiVersion: certificates.k8s.io/v1beta1
+apiVersion: certificates.k8s.io/v1
 kind: CertificateSigningRequest
 metadata:
-  name: ${csrName}
+ name: ${csrName}
 spec:
   groups:
   - system:authenticated
   request: $(cat ${tmpdir}/server.csr | base64 | tr -d '\n')
+  signerName: kubernetes.io/kube-apiserver-client
   usages:
   - digital signature
   - key encipherment
-  - server auth
+  - client auth
```

Comment:
Why do we change  or remove the server auth ?

Reply:
HI @humblec  i have not any exact idea of this, i am following certificate signing request template as describe in [docs](https://kubernetes.io/docs/reference/access-authn-authz/certificate-signing-requests/#create-certificatesigningrequest) . So i have to keep it as server authorisation here? Any help on this would be appreciated. Thanks